packages/react-router-permissions/README.md

```diff
@@ -98,7 +98,82 @@ const authorizationStrategy = (permissions, requirement) => {
 >
 ```
 
-There will be some strategies provided with the package out of the box. Role based and Permissions based strategies will be included.
+There are some strategies provided with the package out of the box. Those are:
+
+- Role based strategy
+
+  ```js
+  const strategy = ['MODERATOR', 'PREMIUM_USER'];
+
+  ...
+  // authorization will pass
+  <AuthorizedRoute
+    path='/authorized-section'
+    requires='MODERATOR'
+  >
+  ...
+
+  ...
+  // authorization will fail
+  <AuthorizedRoute
+    path='/authorized-section'
+    requires='ADMIN'
+  >
+  ...
+  ```
+
+* Permissions based strategy
+
+  ```js
+  const strategy = {
```

Comment:
Those are `permissions` and not strategy. It could also be mentioned which strategy is used for the examples.

Reply:
I will work on examples 👍